caikit_tgis_backend/tgis_backend.py

```diff
@@ -131,74 +136,70 @@ def register_config(self, config: Dict) -> None:
 
     def start(self):
         """Start backend, initializing the client"""
-        self._setup_client()
         self._started = True
 
     def stop(self):
         """Stop backend and unload all models"""
         self._started = False
-        self._client = None
-        self.unload_model("")  # TODO: Clear all models for multi-model
+        for model_id in list(self._model_connections.keys()):
+            log.debug("Unloading model %s on stop", model_id)
+            self.unload_model(model_id)
```

Comment:
unloading here with remote TGIS can have unforeseen consequences I feel, since when stopping this backend, a user may not necessarily want the model to get unloaded on 1 or all remote TGIS's right?

`stop` is also getting called in `__del__` so in that case, would restart in 1 container for example, cause unload of models from TGIS running remotely?